ironfish/src/wallet/database/accountsdb.ts

```diff
@@ -9,6 +9,7 @@ import { Transaction } from '../../primitives/transaction'
 import {
   BigIntLEEncoding,
   BUFFER_ENCODING,
```

Comment:
`BUFFER_ENCODING` is an instance of `BufferEncoding`. Could we do without it?

Reply:
Removed